qiskit_optimization/algorithms/grover_optimizer.py

```diff
@@ -278,10 +292,9 @@ def _measure(self, circuit: QuantumCircuit) -> str:
         probs = self._get_probs(circuit)
         freq = sorted(probs.items(), key=lambda x: x[1], reverse=True)
         # Pick a random outcome.
-        freq[-1] = (freq[-1][0], 1.0 - sum(x[1] for x in freq[0:len(freq) - 1]))
+        freq[-1] = (freq[-1][0], 1.0 - math.fsum(x[1] for x in freq[0:len(freq) - 1]))
```

Comment:
I don't think this line is necessary any more because of the probability normalization in _get_probs. Could you check it?

Reply:
```suggestion
```

Reply:
I checked both of both qasm_simulator and statevector_simulator. It raises no errors without the line now. I think we don't need the line anymore since we normalized the `probs`, so I deleted the line.